File: samples/search/js/delete-track.js
const API_KEY = ""; // Your API key here
const TRACK_ID = "";// Track ID here

var myHeaders = new Headers();
myHeaders.append("Authorization", "Basic " + btoa(API_KEY + ":"));

var requestOptions = {
  method: 'DELETE',
  headers: myHeaders,
  redirect: 'follow'
};

fetch("https://api-us.musiio.com/v1/catalog/track?id=" + TRACK_ID, requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));

File: samples/search/java/DeleteTrack.java
package com.example.search;     // Replace it with your own project group ID

import org.springframework.http.HttpEntity;
import org.springframework.http.HttpHeaders;
import org.springframework.http.HttpMethod;
import org.springframework.http.ResponseEntity;
import org.springframework.web.client.RestTemplate;

public class DeleteTrack {
    private static final String API_KEY = "";       // Your API key
    private static final String URL = "https://api-us.musiio.com/v1/catalog/track?id=";
    public static void main(String[] args) {
        String trackId = args[0];   // input track ID in command: "-Dexec.args='track id here'"
        HttpHeaders headers = HeadersUtils.createHeaders(API_KEY, "");
        RestTemplate restTemplate = new RestTemplate();
        HttpEntity<String> requestEntity = new HttpEntity<>(headers);
        ResponseEntity<String> response = restTemplate.exchange(URL + trackId, HttpMethod.DELETE, requestEntity, String.class);
        System.out.println(response.getBody());
    }
}